graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -76,14 +76,15 @@
 - All names must be valid GraphQL names.
 - Names may not conflict with each other. For instance, you may not rename both "Foo" and "Bar" to
   "Baz". You also may not rename anything to "Baz" if a type "Baz" already exists and is not also
-  being renamed or suppressed.
-- Special rules apply to the renamings argument if it's iterable (e.g. if a dict). If iterable,
-  renamings may not contain no-op entries. In other words:
-    - A string type_name may be in renamings only if there exists a type in the schema named
-      type_name (since otherwise that entry would not affect any type in the schema).
-    - A string type_name is in renamings, then renamings[type_name] != type_name (since applying the
-      renaming would not change the type named type_name).
-  If not iterable, then these no-op rules don't apply.
+  being renamed or suppressed. The same rules apply for fields that belong to the same type, since
+  they share a namespace as well.
+- There exist special rules for iterable renamings (e.g. if renamings are represented as a
+  dictionary)-- specifically, no-op renamings are not allowed.
+  - If type_renamings argument is iterable:
```

Comment:
```suggestion
- When the renaming argument is an iterable value (e.g., a dict), a few additional rules apply:
```
Then de-indent the below bullet points. This says the same thing more succinctly and with one fewer level of indentation.

Reply:
This additional layer is actually part of the `renamings` -> `type_renamings` refactor, since when we get field renamings it'd be helpful to distinguish between `type_renamings`, field renamings for a particular type (which could be a `Dict[str, str]`), and field renamings in general (which could be, among other things, a `Dict[str, FieldRenamingsForParticularType]`)

Reply:
I agree with Leon's choice here.